tests/arch/xilinx/add_sub.ys

```diff
@@ -7,5 +7,5 @@ cd top # Constrain all select calls below inside the top module
 select -assert-count 14 t:LUT2
 select -assert-count 6 t:MUXCY
 select -assert-count 8 t:XORCY
-select -assert-none t:LUT2 t:MUXCY t:XORCY %% t:* %D
+select -assert-none t:LUT2 t:MUXCY t:XORCY t:IBUF t:OBUF %% t:* %D
```

Comment:
It might be slightly cleaner if you change these tests to call `-noiopad` (since they *are* supposed to test small out of context circuits)